eurocropsml/acquisition/region.py

```diff
@@ -40,129 +40,142 @@ def add_nuts_regions(
         "statistical-units/territorial-units-statistics"
     )
 
-    label_dir = final_output_dir.joinpath("labels")
-    geom_dir = final_output_dir.joinpath("geometries")
+    for month in range(config.months[0], config.months[1] + 1):
 
-    if not (
-        label_dir.joinpath(f"{config.country}_labels.parquet").exists()
-        and final_output_dir.joinpath(f"{config.country}.parquet").exists()
-        and geom_dir.joinpath(f"{config.country}.geojson").exists()
-    ):
-        shapefile: gpd.GeoDataFrame = pyogrio.read_dataframe(shape_dir)
+        label_dir = final_output_dir.joinpath(f"{month}", "labels")
+        geom_dir = final_output_dir.joinpath(f"{month}", "geometries")
```

Comment:
In the clipper module we used the paths `f"clipper/{month}"`, so functionality followed by the month, here it is the other way around. Is there a specific reason to do it this way? (Not a big deal, but seems a bit inconsistent ;) )

Reply:
Each month has it's own parquet file, so we need to have a month division in the raw_data directory